cmd/download.go

```diff
@@ -74,34 +74,44 @@ func get(src, dest string, mode getter.ClientMode, cleanGitFolder bool) error {
 
 	logrus.Debugf("complete url downloading: %s -> %s", src, dest)
 
+	var tempDest = dest + ".tmp"
+
 	pwd, err := os.Getwd()
 	if err != nil {
 		return err
 	}
 
 	client := &getter.Client{
 		Src:  src,
-		Dst:  dest,
+		Dst:  tempDest,
 		Pwd:  pwd,
 		Mode: mode,
 	}
 
 	logrus.Debugf("let's get %s -> %s", src, dest)
```

Comment:
this debug log will be misleading, because it will actually be downloaded to `tmpDest` instead of `dest`.

Reply:
Will do 👍